joefetch.py

```diff
@@ -1,22 +1,65 @@
 import platform
 import psutil
 import time
+import random
 
 def print_logo():
-    logo = '''
-                                
-                              
-      $$\  $$$$$$\   $$$$$$\  
-      \__|$$  __$$\ $$  __$$\ 
-      $$\ $$ /  $$ |$$$$$$$$ |
-      $$ |$$ |  $$ |$$   ____|
-      $$ |\$$$$$$  |\$$$$$$$\ 
-      $$ | \______/  \_______|
-$$\   $$ |                    
-\$$$$$$  |                    
- \______/                     
+    logotype = random.randint(0,4)
```

Comment:
This is adding new logos. It also fixes a weird error. This is what the r infront of `logo = r''' ` is for.